README.md

```diff
@@ -35,33 +35,28 @@ request.
 ## Branches
 
 * Choose *short* and *descriptive* names:
+* Namespace your branches with your username
```

Comment:
Why?

Reply:
Rather, i'd enjoy seeing your why in the guide.

Reply:
1. it prevents any collisions with branches; ie we both name a branch `css-fixes` we don't collide if they are `rauhryan/css-fixes` and `drusellers/css-fixes`
2. it's a nice to have in GUI's that folder them
![bonsai__branch__robsears-sm-patch-4__and_adds_omc_opinions_to_the_style_guide_by_rauhryan_ _pull_request__1_ _omc_git-style-guide](https://user-images.githubusercontent.com/68954/39015468-0d12cbfe-43e3-11e8-8072-d0a77956f67e.png)